helm/easeprobe/templates/deployment.yaml

```diff
@@ -41,7 +41,8 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
-            - containerPort: 8181
+            - name: http
```

Comment:
Consider using a variable for the port name instead of hardcoding it. This would allow for more flexibility if the port name needs to be changed in the future. [medium]